docs/dev/analysis/dml-line.rst

```diff
@@ -88,6 +88,27 @@ MS API
   | msoLineSquareDot      | 2     | Line is made up of square dots.  |
   +-----------------------+-------+----------------------------------+
 
+* | MsoArrowheadStyle Enumeration
+  | https://docs.microsoft.com/en-us/office/vba/api/Office.MsoArrowheadStyle
+
+  +-----------------------+-------+----------------------------------+
+  | Name                   | Value | Description                     |
+  +-----------------------+-------+----------------------------------+
+  | msoArrowheadDiamond    | 5     | Diamond-shaped.                 |
+  +-----------------------+-------+----------------------------------+
+  | msoArrowheadNone       | 1     | No arrowhead.                   |
+  +-----------------------+-------+----------------------------------+
+  | msoArrowheadOpen       | 3     | Open.                           |
+  +-----------------------+-------+----------------------------------+
+  | msoArrowheadOval       | 6     | Oval-shaped.                    |
+  +-----------------------+-------+----------------------------------+
+  | msoArrowheadStealth    | 4     | Stealth-shaped.                 |
+  +-----------------------+-------+----------------------------------+
+  | msoArrowheadStyleMixed | -2    | Return value only.              |
+  +-----------------------+-------+----------------------------------+
+  | msoArrowheadTriangle   | 5     | Triangular.                     |
+  +-----------------------+-------+----------------------------------+
+
 
 Specimen XML
 ------------
```

Comment:
The schema definitions need an entry for `CT_LineEndProperties` and the simple types (ST_ definitions) it uses. You can find those here:  
https://github.com/scanny/python-pptx/blob/e558a3870815b496de5eddad6d5f49983e8d8c3c/spec/ISO-IEC-29500-4/xsd/dml-main.xsd#L2228

Reply:
I'd like to see a line-end focused pair of specimen XML. One should show the default (assumedly with no `a:headEnd` or `a:tailEnd` elements) and at least one with all line-end attributes set (style, length, width I believe).